Gemfile

```diff
@@ -29,6 +29,7 @@ gem 'i18n', '~> 0.6.4'
 gem 'rails-i18n', '~> 3.0.0'
 gem 'turbolinks', '~> 2.5'
 gem 'logging', '>= 1.8.0', '< 3.0.0'
+gem 'roadie', '~> 2.4'
```

Comment:
Looks like roadie-rails is the right gem to be using to use roadie 3.x with Rails.